### PR TITLE
docs: Tweak TOC.yml structure

### PR DESCRIPTION
**Pull request by lukeblevins**
GitHub Issue (If applicable): N/A

## PR Type
- Documentation content changes

What kind of change does this PR introduce?

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tested code with current [supported SDKs](../README.md#supported)
- [x] Docs have been added/updated which fit [documentation template](https://github.com/nventive/Uno/blob/master/doc/.feature-template.md). (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](doc/articles/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] [Wasm UI Tests](doc/articles/working-with-the-samples-apps.md#running-the-webassembly-ui-tests-snapshots) are not showing unexpected any differences. Validate PR `Screenshots Compare Test Run` results.
- [x] Contains **NO** breaking changes
- [ ] Updated the [Release Notes](https://github.com/nventive/Uno/tree/master/doc/ReleaseNotes)
- [ ] Associated with an issue (GitHub or internal)

## Other information

Internal Issue (If applicable): N/A

### CODE REVIEW

File: doc/00-Introduction.md [hunk 1]
```diff
@@ -2,9 +2,9 @@
 
 ## Overview
 
-The unoapp-extensions template uses Uno.Extensions.Hosting to configure the hosting environment for the application. This includes loading configuration information, setting up a dependency container and configure logging for the application. The template is already preconfigured so all you need to do is add your project dependencies, such as services for loading data, and register the routes for your application. 
+The unoapp-extensions template uses Uno.Extensions.Hosting to configure the hosting environment for the application. This includes loading configuration information, setting up a dependency container and configure logging for the application. The template is already pre-configured so all you need to do is add your project dependencies, such as services for loading data, and register the routes for your application.
```
Comment (agneszitte-nventive) [inline doc/00-Introduction.md:5]:
```suggestion
The `unoapp-extensions` template uses Uno.Extensions.Hosting to configure the hosting environment for the application. This includes loading configuration information, setting up a dependency container, and configuring logging for the application. The template is already pre-configured so all you need to do is add your project dependencies, such as services for loading data, and register the routes for your application.
```

File: doc/00-Introduction.md [hunk 2]
```diff
@@ -2,9 +2,9 @@
...
 
-The template is a typical Uno application that makes used of a Shared project for all the application logic and corresponding platform specific projects for each supported platform. The purpose of this template is to provide a starting point for real world applications, making it easy to generate production ready applications. 
+The template is a typical Uno application that makes used of a Shared project for all the application logic and corresponding platform specific projects for each supported platform. The purpose of this template is to provide a starting point for real world applications, making it easy to generate production ready applications.
```
Comment (agneszitte-nventive) [inline doc/00-Introduction.md:7]:
```suggestion
The template is a typical Uno application that makes use of a Shared project for all the application logic and corresponding platform-specific projects for each supported platform. The purpose of this template is to provide a starting point for real-world applications, making it easy to generate production-ready applications.
```

File: doc/00-Introduction.md [hunk 3]
```diff
@@ -17,11 +17,9 @@ In addition to referencing Uno, the Guidance Template also references the follow
...
-| Uno.Extensions.Reactive      | Developement framework for reactive applications                                                          |
+| Uno.Extensions.Reactive      | Development framework for reactive applications                                                          |
 | Uno.Extensions.Serialization | Helpers for json serialization                                                                            |
 
 ## Uno.Extensions
 
-The Uno.Extensions is a series of packages designed to encapsulate common developer tasks associated with building cross platform mobile, desktop and web applications using the Uno platform. 
-
-
+The Uno.Extensions is a series of packages designed to encapsulate common developer tasks associated with building cross platform mobile, desktop and web applications using the Uno platform.
```
Comment (agneszitte-nventive) [inline doc/00-Introduction.md:25]:
```suggestion
The Uno.Extensions is a series of packages designed to encapsulate common developer tasks associated with building cross-platform mobile, desktop, and web applications using the Uno platform.
```

File: doc/00-Introduction.md [hunk 4]
```diff
@@ -17,11 +17,9 @@ In addition to referencing Uno, the Guidance Template also references the follow
 | Uno.Extensions.Localization  | Provides access to localizable resources                                                                  |
 | Uno.Extensions.Logging       | Configures application logging                                                                            |
 | Uno.Extensions.Navigation    | Routing framework for frame based navigation                                                              |
```
Comment (agneszitte-nventive) [inline doc/00-Introduction.md:19]:
```suggestion
| Uno.Extensions.Navigation    | Routing framework for frame-based navigation                                                              |
```

File: doc/Extensions/Configuration/HowTo-Configuration.md [hunk 5]
```diff
@@ -0,0 +1,7 @@
+# How-To: Configuration
+
+Configuration in Uno.Extensions provides you with a consistent way to view configuration data from a number of distinct sources. This pattern is not intended to allow for programmatic writes by itself, but instead offers a single representation of any number of providers like `appsettings.json`.
```
Comment (agneszitte-nventive) [inline doc/Extensions/Configuration/HowTo-Configuration.md:3]:
```suggestion
Configuration in Uno.Extensions provide you with a consistent way to view configuration data from a number of distinct sources. This pattern is not intended to allow for programmatic writes by itself, but instead offers a single representation of any number of providers like `appsettings.json`.
```

File: doc/Extensions/GettingStarted/HowTo-UsingUnoExtensions.md [hunk 6]
```diff
@@ -6,43 +6,42 @@ The Uno.Extensions follows the Microsoft.Extensions model that creates a host en
...
 ### 1. Installing extension templates
 
-The `dotnet` templates included in the `Uno.Extensions.Templates` package are used to easily create new projects that already reference the Uno.Extensions. 
+The `dotnet` templates included in the `Uno.Extensions.Templates` package are used to easily create new projects that already reference the Uno.Extensions.
 
 * Open a command prompt and run the following
 
     `dotnet new -i Uno.Extensions.Templates`
 
 * Navigate to the desired projects directory, and use the unoapp-extensions template to generate the starter solution discussed above
```
Comment (agneszitte-nventive) [inline doc/Extensions/GettingStarted/HowTo-UsingUnoExtensions.md:23]:
```suggestion
* Navigate to the desired projects directory, and use the `unoapp-extensions` template to generate the starter solution discussed above
```

File: doc/Extensions/GettingStarted/HowTo-UsingUnoExtensions.md [hunk 7]
```diff
@@ -68,16 +67,16 @@ The generated solution will contain:
 
 * Press Ctrl + F5 to start the WASM project without debugging.
 
-* Once the application is compiled, it will launch inside your default browser. Take note of the URL which should look something like this: https://localhost:11111/
+* Once the application is compiled, it will launch inside your default browser. Take note of the URL which should look something like this: <https://localhost:11111/>
```
[the remark this answers is not in the document]
Comment (lukeblevins) [reply doc/Extensions/GettingStarted/HowTo-UsingUnoExtensions.md:70]: No reason :) This was an error with the linting extension that was in use.

File: doc/tutorials-intro.md [hunk 8]
```diff
@@ -0,0 +1,23 @@
+# Solve Specific Problems with Uno.Extensions
+
+This section gathers how-tos, advanced tutorials, and detailed explainers that will help you use Uno.Extensions to solve specific problems.
```
[the remark this answers is not in the document]
Comment (lukeblevins) [reply doc/tutorials-intro.md:3]: @Xiaoy312 I'm not sure this one sounds concise enough, but I've incorporated your other feedback. Much appreciated!

**Review by nickrandolph — APPROVED** [review]

**Pull request merged**